tests/tex-output/commentary/commentary.tex

```diff
@@ -18,16 +18,19 @@
 ]{Alegreya}
 \newcommand\commhead[2]{\grecommentary{#1}\grecommentary{#2}}
 \begin{document}
-\gresetheadercapture{x-commentary}{grecommentary}{}%
+\gresetheadercapture{commentary}{grecommentary}{}%
 \gregorioscore[a]{commentary}
 
-\gresetheadercapture{x-commentary}{grecommentary}{string}%
+\gresetheadercapture{commentary}{grecommentary}{string}%
 \gregorioscore[a]{commentary}
 
-\gresetheadercapture{x-commentary}{commhead}{name}%
+\gresetheadercapture{commentary}{}{}%
```

Comment:
Instead of putting a new test here, it would be better to simply move this line down to immediately precede the spacing test.  I've come to realize that as is the spacing test is not exercising the optional argument.